_pages/extension_sharing_strategies.md

```diff
@@ -0,0 +1,143 @@
+
+#### 2.1.1 [ndx-template](https://github.com/nwb-extensions/ndx-template): Extension template
+
+The [ndx-template](https://github.com/nwb-extensions/ndx-template) provides a standard Git template repository with a standard setup for creating and sharing extensions and instructions for how to use the template. Using [Cookiecutter](https://cookiecutter.readthedocs.io/en/latest/), users can easily modify the template to create the basic setup for their ndx-template repository. See the [ndx-template](https://github.com/nwb-extensions/ndx-template) repository for details on the organization of template.
+
+#### 2.1.2 [staged-extensions](https://github.com/nwb-extensions/staged-extensions): Creating custom ndx recipes
+
+The [staged-extensions](https://github.com/nwb-extensions/staged-extensions) repository is a place to submit NWB:N extension recipes before they become fully fledged NWB extensions.
+
+#### 2.2.3 ndx-custom-recipe: Custom extension catalog recipe
+
+The NDX Catalog contains a collection of ndx-custom-recipes created by the creators of the extensions. See below for further details.
+
+
+### 2.3 Custom Extension Code Repositories
+
+#### 2.3.1 ndx-custom-recipe: Custom extension catalog recipe
+
+The ndx-custom-recipe is created from the staged-extensions repository as part of the NDX catalog and MUST have the same name as the main namespace of the NDX. This strategy helps ensure that there are no name conflicts between public NDX namespace names.  The ndx-custom-recipy  repository MUST contain the following:
+
+* **/** : root directory of the repo
+   * **/ndx-meta.yaml**: REQUIRED YAML file with the metadata describing the extensions. The YAML file MUST contain a dict with the following keys:
+      * **src: <URL>** : URL to the public repository with the sources of the extension
+      * **pip: <URL>** : URL for installing the extensions from PyPi
+   * **/README.(md|rst)** : REQUIRED readme with basic description of the extension, badges for CI, install instructions etc.
+   * **/review**: OPTIONAL folder with RST documents related to the review of the extension for acceptance as part of the NWB:N core schema. If present the /review folder MUST contain:
+      * **/review/Makefile** : Standard Sphinx makefile for building the review docs  in HTML, LATEX etc.
+      * **/review/make.bat** : Standard Sphinx make.bat file
+      * **/review/README.md** : Standard readme file with instructions on how to build and use the docs .
+      * **/review/source** : Folder with the RST sources for the Sphinx review documents
+         * The /review/source folder MUST contain:
+            * **/docs/source/conf.py**: Sphinx configuration file for building the docs.
```

Comment:
It feels like overkill to have the review documents by rendered via sphinx. We can just use plain markdown?

Reply:
I agree, having the review documents as markdown instead of Sphinx is probably fine. For the actual documentation of the extension itself we should stick with Sphinx but for the review documents (which are only there to document that community review process and discussions), markdown should be sufficient.

Reply:
I updated the text accordigly to change the review docs to Markdown and separate the OPTIONAL from the REQUIRED parts